src/libs/ReportActionsUtils.ts

```diff
@@ -109,6 +109,22 @@ function isChannelLogMemberAction(reportAction: OnyxEntry<ReportAction>) {
     );
 }
 
+/**
+ * Returns whether the report is dequeued only using reportID
+ */
+function isReimbursementDeQueued(reportID: string): boolean {
+    if (!allReportActions) {
+        return false;
+    }
+
+    const reportAction = Object.values(allReportActions[`${reportID}`] ?? {});
```

Comment:
Do we need to wrap `reportID` with string literal? reportID should already be a string